besu/src/main/java/org/hyperledger/besu/cli/subcommands/ValidateConfigSubCommand.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.cli.subcommands;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.hyperledger.besu.cli.subcommands.ValidateConfigSubCommand.COMMAND_NAME;
+
+import org.hyperledger.besu.cli.BesuCommand;
+import org.hyperledger.besu.cli.DefaultCommandValues;
+import org.hyperledger.besu.cli.util.TomlConfigFileDefaultProvider;
+
+import java.io.PrintStream;
+import java.nio.file.Path;
+
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ParentCommand;
+
+@Command(
+    name = COMMAND_NAME,
+    description = "This command provides Besu config validation.",
```

Comment:
Just to clarify: this is only syntax validation. We could still use mutually incompatible flags/config values, pass in this verification, and then fail during app startup?

I am not saying we should change it, but I do wonder if we could describe the option more accurately. _This command provides Besu config **syntax** validation._ would not be helpful, though.

Reply:
this.out.print("TOML config file is valid on basic inspection. Further dependencies between related options is checked when Besu starts.");

Reply:
is that better?

Reply:
Will not win awards for brevity; but yes, it is better.